airbyte-integrations/bases/base-python/base_python/sdk/streams/rate_limiting.py

```diff
@@ -0,0 +1,58 @@
+import sys
+import time
+
+import backoff
+from ...logger import AirbyteLogger
+from requests import exceptions
+from .exceptions import DefaultBackoffException, CustomBackoffException
+
+TRANSIENT_EXCEPTIONS = (DefaultBackoffException, exceptions.ConnectTimeout, exceptions.ReadTimeout)
+
+# TODO inject singleton logger?
+logger = AirbyteLogger()
```

Comment:
use standard logger, see my comment in https://github.com/airbytehq/airbyte/issues/1279